src/test/resources/omop/settings.json

```diff
@@ -170,37 +170,111 @@
+          "fpk_measurement_unit",
+          "fpk_measurement_concept_s",
+          "fpk_measurement_value",
+          "fpk_measurement_type_concept",
+          "fpk_measurement_operator"]
+      }
     },{
       "name": "Visit",
-      "values": []
+      "table": {
+        "datasetTableName": "visit_occurrence",
+        "columns": [],
+        "primaryTableRelationship": "fpk_person_visit",
+        "secondaryTableRelationships": [
+          "fpk_visit_preceding",
+          "fpk_visit_concept_s",
+          "fpk_visit_type_concept",
+          "fpk_visit_concept",
+          "fpk_visit_discharge"]
+      }
     },{
       "name": "Device",
-      "values": []
+      "table": {
+        "datasetTableName": "device_exposure",
+        "columns": [],
+        "primaryTableRelationship": "fpk_person_device",
+        "secondaryTableRelationships": [
+          "fpk_device_concept",
+          "fpk_device_concept_s",
+          "fpk_device_type_concept"]
+      }
     },{
       "name": "Procedure",
-      "values": []
+      "table": {
+        "datasetTableName": "procedure_occurrence",
+        "columns": [],
+        "primaryTableRelationship": "fpk_person_procedure",
+        "secondaryTableRelationships": [
+          "fpk_procedure_concept",
+          "fpk_procedure_concept_s",
+          "fpk_procedure_type_concept",
+          "fpk_procedure_modifier"]
+      }
     },{
       "name": "Condition",
-      "values": []
+      "table": {
+        "datasetTableName": "condition_occurrence",
+        "columns": [],
+        "primaryTableRelationship": "fpk_person_condition",
+        "secondaryTableRelationships": [
+          "fpk_condition_concept",
+          "fpk_condition_type_concept",
+          "fpk_condition_status_concept",
+          "fpk_condition_concept_s"]
+      }
     },{
       "name": "Observation",
-      "values": []
+      "table": {
+        "datasetTableName": "observation",
+        "columns": [],
+        "primaryTableRelationship": "fpk_person_observation",
+        "secondaryTableRelationships": [
+          "fpk_observation_period_person",
+          "fpk_observation_concept",
+          "fpk_observation_concept_s",
+          "fpk_observation_unit",
+          "fpk_observation_qualifier",
+          "fpk_observation_type_concept",
+          "fpk_observation_period_concept",
+          "fpk_observation_value"]
+      }
     },{
-      "name": "Person",
-      "values": []
-    }
-  ],
-  "datasetConceptSets": [
-    {
       "name": "Demographics",
-      "featureValueGroupName": "Person"
+      "table": {
+        "datasetTableName": "person",
+        "columns": [],
+        "secondaryTableRelationships": ["fpk_person_ethnicity_concept", "fpk_person_ethnicity_concept_s", "fpk_person_race_concept"]
+      }
+    },{
+      "name": "Genomics",
+      "table": {
+        "datasetTableName": "sample",
+        "columns": [],
+        "primaryTableRelationship": "fpk_person_sample"
+      }
     }
```

Comment:
I think we had removed the Genomics set because there wasn't a "sample" dataset. I think this would break snapshot creation if genomics was selected in the request.

Reply:
Oh okay good catch!